### ISSUE TITLE
error.response.statusCode doesn't exist

### ISSUE THREAD

**Issue by frankenthumbs** [Microsoft/appcenter-cli#393]
hey y'all,

I'm trying to run appcenter-cli on 1.0.11 but getting an error. seemingly with the statusCode in the response.

has anybody seen this before/know what's going on?

Thank you in advanced.

```
Command Failure at TypeError: Cannot read property 'statusCode' of undefined
    at CodePushReleaseCommand.<anonymous> (/Users/jenkins/workspace/app/node_modules/appcenter-cli/dist/commands/codepush/lib/release-command-skeleton.js:69:36)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/jenkins/workspace/app/node_modules/appcenter-cli/dist/commands/codepush/lib/release-command-skeleton.js:10:58)
    at <anonymous>
Error: Command 'codepush release --target-binary-version=^0.0.0 --deployment-name=Production --app=appName --update-contents-path=/Users/jenkins/workspace/app/ios/main.jsbundle --token=token --debug' failed with exception "Cannot read property 'statusCode' of undefined"
```

**Comment by frankenthumbs** [comment]
i believe this is the root cause. https://github.com/Microsoft/appcenter-cli/pull/389

**Comment by iTOYS** [comment]
Hi @frankenthumbs 
Mentioned above issue may be cause of your error. We'll merge solution soon.
For now you can avoid error by adding `--env prod` flag to command.
Also I've created [PR](https://github.com/Microsoft/appcenter-cli/pull/405) for correct error handling.

Thanks!

**Comment by amchew** [comment]
Hi @frankenthumbs , this issue is resolved with @iTOYS 's PR #405, and released in the new CLI v1.1.0 today. Could you please run the new CLI version and see if this issue is resolved? 

Also special shout out to @iTOYS for making the PR!

**Comment by amchew** [comment]
Hi @frankenthumbs , this issue is resolved with @iTOYS 's PR #405. Please test this out, and let us know if you have any questions. Thanks again to @iTOYS (🥇) for making the PR!

I'll be closing this ticket for now.
Amanda
